### ISSUE TITLE
ProjectName_BackUpThisFolder_ButDontShipItWithYourGame included in package

### ISSUE THREAD

**Issue by MonsterArtur1** [microsoft/gdk-unity-package#57]
A directory called ProjectName_BackUpThisFolder_ButDontShipItWithYourGame is generated inside build folder automatically by unity. Inside there are all pdbs from il2cpp project.  (here is unity doc [https://docs.unity3d.com/Manual/WindowsPlayerIL2CPPBuildProducedFiles.html?_ga=2.74103953.2055947039.1598316643-1897641508.1594776991](url))

The default build script should move this directory, before creating layout.xml file (or ignore this dir). Now layout.xml includes those files in package.

**Comment by payzer** [comment]
Thank you for pointing that out. We'll look into fixing that.

**Comment by lassade** [comment]
I solved this by including 2 steps before and after packing calling `DebugSymbols(true)` and `DebugSymbols(false)` respectively

```
bool DebugSymbols(bool removeOrInclude)
        {
            try
            {
                var backupFolderPath = $"{Application.productName}_BackUpThisFolder_ButDontShipItWithYourGame";
                var originalFolderPath = @$"{buildOutputFolderPath}\{Win32OutputDirectory}\{backupFolderPath}";
                var tempFolderPath = @$"{buildOutputFolderPath}\{backupFolderPath}";
                if (removeOrInclude)
                {
                    Directory.Move(originalFolderPath, tempFolderPath);
                }
                else
                {
                    Directory.Move(tempFolderPath, originalFolderPath);
                }
                return true;
            }
            catch (Exception e)
            {
                UnityEngine.Debug.LogException(e);
                return false;
            }
        }
```

**Comment by payzer** [comment]
Thank you for sharing your solution. I'm sure it will be useful to many others and us as we look to solve it in the plugin.

**Comment by tculotta** [comment]
Thanks for bringing this to our attention and sharing your solution. This has now been fixed here #72 included in the [most recent release](https://github.com/microsoft/gdk-unity-package/releases/tag/2104.0.20220209).
